doc/api/readline.md

```diff
@@ -7,22 +7,53 @@
 <!-- source_link=lib/readline.js -->
 
 The `readline` module provides an interface for reading data from a [Readable][]
-stream (such as [`process.stdin`][]) one line at a time. It can be accessed
-using:
+stream (such as [`process.stdin`][]) one line at a time.
 
-```js
+To use the promise-based APIs:
+
+```mjs
+// Using ESM Module syntax:
```

Comment:
```suggestion
// Using ES Modules syntax:
```

Reply:
This was copied from `fs.md`, but since it has been removed in e8b086228a627adaa6b8bcb7fc47b7574091543c, I'm going to remove the comment instead.